visualization/app/codeCharta/e2e/url.e2e.ts

```diff
@@ -38,7 +38,7 @@ describe("codecharta", () => {
 
 	async function handleErrorDialog() {
 		const message = await dialogError.getMessage()
-		expect(message).toEqual("One or more files from the given file URL parameter could not be loaded. Loading sample files instead.")
+		expect(message).toEqual("One or more files from the given file URL parameter could not be loaded. Loading sample files instead. undefined")
```

Comment:
Is this actually a reasonable test? Printing `undefined` at the end is not providing any useful information.
Can this happen in the application during runtime?

Reply:
Not really. The error should always have a message.

Reply:
We should find out why we can't simulate this in our tests. This test is not useful like this and will confuse people in the future.

Reply:
In theory, yes. It should already be an improvement in the average case though. Should I add a TODO comment?

Reply:
Yes, please. I really want to merge this branch.